chia/util/file_keyring.py

```diff
@@ -243,237 +254,146 @@ def on_modified(self, event: Union[FileSystemEvent, DirModifiedEvent]) -> None:
         self.check_if_keyring_file_modified()
 
     def check_if_keyring_file_modified(self) -> None:
-        if self.keyring_path.exists():
```

Comment:
Unfortunately I don't remember why this check was here. It may have been to prevent hitting some exception beyond `FileNotFoundError`, possibly specific to a single platform. Anyway, hopefully the tests will sufficiently cover any corner cases that this code may have handled.

Reply:
This belongs into #12669